tg.go

```diff
@@ -239,6 +239,11 @@ func tgLoadContacts(tg *tgclient.TGClient) (*mtproto.TL_contacts_contacts, error
 	return &contacts, nil
 }
 
+func tgLogout(tg *tgclient.TGClient) () {
+	tg.SendSyncRetry(mtproto.TL_auth_logOut{}, time.Second, 0, 30*time.Second)
```

Comment:
It should check for error since the request may be not always successful. Something like
```go
res := tg.SendSyncRetry(mtproto.TL_auth_logOut{}, time.Second, 0, 30*time.Second)
if _, ok := res.(mtproto.TL_auth_loggedOut); !ok {
  return merry.New(mtproto.UnexpectedTL("logout", res))
}